docs/contrib/PlatformBringUp.md

```diff
@@ -323,7 +342,7 @@ The tapcell configuration file is used to insert tapcells and endcaps into the d
 ## setRC Configuration
 
 
-setRC allows the user to define resistances and capacitances for layers and vias using the `set_layer_rc` command. There is also a command that allows you to set the resistance and capacitance of routing wires using the `set_wire_rc`. Following is a generic example of a setRC configuration file which sets the resistance and capacitance of five metal layers, four vias, one signal wire, and one clock wire.
+setRC allows the user to define resistances and capacitances for layers and vias using the `set_layer_rc` command. There is also a command that allows you to set the resistance and capacitance of routing wires using the `set_wire_rc`. Often, per-unit-length values are available in the PDK user guide. For `set_layer_rc`, Liberty units need to be used. Following is a generic example of a setRC configuration file which sets the resistance and capacitance of five metal layers, four vias, one signal wire, and one clock wire.
```

Comment:
The statement "Often, per-unit-length values are available in the PDK user guide" is kind of floating, maybe clarify that those are the units the set_layer_rc is expecting.

Reply:
Clarified that `set_layer_rc` is expecting per-unit-length values and that they can often be found in the PDK user guide.